packages/eslint-plugin-office-addins/src/rules/call-sync-after-load.ts

```diff
@@ -68,10 +68,14 @@ export = {
         }
 
         if (operation === "Read" && variable) {
-          const propertyName: string = findPropertiesRead(
+          let propertyName: string = findPropertiesRead(
             reference.identifier.parent
           );
 
+          if (needSync.has({ variable: variable.name, property: "*" })) {
+            propertyName = "*";
+          }
```

Comment:
I don't think we need to update the propertyName value like this . . . we just need do the test for variable.name + "*" in the next if statement (using '||').

Reply:
That what I did before, the problem was that later we report the error based on `propertyName`. So if we just check for the existence of `*` the error message would be with a wrong `propertyName`.

Reply:
I don't think you want to print out the '*'.  That would make the message "Call context.sync() after calling load on 'obj' for property '*' and before reading the property." . . . but there is no property '*', that just happens to be what the load call argument it.

Reply:
The message would be as you said and I think it is more correct than without it. For example, in the following code:
```
var range = worksheet.getSelectedRange();
range.load("*");
console.log(range.address);
```
Would we rather the message to be: 
1. "Call context.sync() after calling load on range for property * and before reading the property."
2. "Call context.sync() after calling load on range for property address and before reading the property."

I prefer number 1, given that's what the load is. We can remove the "property" in the message to also give room to mention star, which is not a property.

Reply:
I think 2 is better because it identifies 'address' as the property that needs to be sunk where as * is ambiguous.

And actually after reading it now I have a few new thoughts:
- We could leave the name of the property out of the message so it just reads "Call context.sync() after calling load and before reading the property."
- The wording of the sentence in general it a little confusing . . . the phrase "calling load on range for property address" could be miss understood.
- We should probably get PM and or doc writers to get the right wording for our message . . . they are better at it then we are :-).

Reply:
Sure, talking to someone is probably the right approach. I will email Ricky about this example I provided here to see what he suggests.

Reply:
@millerds I just fixed the message!
The PR is ready to be reviewed again!